### PR TITLE
Created status page

### PR DESCRIPTION
**Pull request by Nolski**

### CODE REVIEW

File: airmozilla/popcorn/static/popcorn/js/editor.js [hunk 1]
```diff
@@ -27,11 +27,7 @@ $(document).ready(function () {
             csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val()
         })
         .done(function (response) {
-            var message = "Your edit has been saved and sent it for "
-            + "transcoding. Once it completes, which can take several minutes, "
-            + "we'll automatically update the event to use your latest "
-            + "video edits.";
-            alert(message);
+            document.location = '/popcorn/status/' + $('#editor').data('slug');
```
Comment (peterbe) [inline airmozilla/popcorn/static/popcorn/js/editor.js]: Due to the way Popcorn works, I think we should put an alert here of our own. 
Also, what can we do to prevent Popcorn throwing up that "Leave page?" dialog? How about we use jQUery to destroy the iframe. Do you think that'd work?

File: airmozilla/popcorn/static/popcorn/js/status.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        event_id: $('#revisions').data('event'),
+        csrfmiddlewaretoken: $('input[name="csrfmiddlewaretoken"]').val(),
+    })
+    .done(function (data) {
+        console.log('Done!', data);
+    })
+    .fail(function (data) {
+        // error
+    });
+}
```
Comment (peterbe) [inline airmozilla/popcorn/static/popcorn/js/status.js]: Is this file still needed?

**Comment by peterbe** [comment]
We need tests.

**Comment by peterbe** [comment]
So we need to get this landed. I can't upgrade to django 1.7 otherwise. I'll try to help by making some counter-PRs.

**Comment by Nolski** [comment]
Let me take a look at this today. Been caught up this last week with housing and stuff.

**Comment by Nolski** [comment]
@peterbe rebased and squashed. If tests pass this should be good to merge with your approval
